docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>Same here! Generally try to avoid this kind of <code>print</code>  in notebooks... take advantage of those __html__repr__'s!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I'm not sure what you mean by this.  Can you provide an example?

Reply:
Instead of putting this single large print call in a single cell, notebooks are better suited for just

```
>>> variable
value_of_variable
>>> other variable
value of other variable
```

It's cleaner that way, and if `value_of_variable` defines how to represent itself in HTML outputs, it'll render better in notebooks. That's true for Quantities, for example.

Reply:
Ahh...that makes more sense. I thought you were talking about embedding it in the markdown cell.

I generally don't like using `print` in notebooks either. I'll see if your suggested way looks cleaner.